src/main.qml

```diff
@@ -270,75 +269,77 @@ ApplicationWindow {
                     text: qsTr("Keyboard navigation: <tab> navigate to next button <space> press button/select item <arrow up/down> go up/down in lists")
                 }
 
-                RowLayout {
+                Rectangle {
```

Comment:
This section looks MUCH more complicated in the GitHub diff than it actually is.  If you use VSCode (or the Github.dev online one), you'll see that it's just a minor move.